backend/sublet/serializers.py

```diff
@@ -61,6 +61,36 @@ class SubletSerializer(serializers.ModelSerializer):
         many=True, queryset=Amenity.objects.all(), required=False
     )
 
+    def validate_publish(self, validated_data, instance=None):
+        fields = [
+            "title",
+            "address",
+            "price",
+            "negotiable",
+            "start_date",
+            "end_date",
+            "expires_at",
+        ]
+
+        newest_fields = [
+            (
+                field,
+                (
+                    validated_data[field]
+                    if field in validated_data
+                    else getattr(instance, field, None)
+                    if instance
+                    else None
+                ),
```

Comment:
technically i think u can also do `validated_data.get(field) or getattr(instance, field, None)`

Reply:
<img width="920" alt="image" src="https://github.com/pennlabs/penn-mobile/assets/71510289/1365ca83-db32-4668-a9a5-395a67440f08">


Yep, confirmed! I think I like this better can we perhaps change it to that